### PR TITLE
Updated the openshift-host-builder with correct versions of packages

### PR DESCRIPTION
**Pull request by oybed**
#### What does this PR do?

Updating the openshift-hostbuilder image to work with the correct Ansible version + correct OSP repos.
#### How should this be manually tested?

Spin up the docker container and verify that the python-*client RPMs are installed and tools available + verify that ansible is version 1.9.x, e.g.:

```
>> ansible --version
ansible 1.9.4
```
#### Is there a relevant Issue open for this?

N/A
#### Who would you like to review this?

/cc @sabre1041 @etsauer

### CODE REVIEW

File: docker/openshift-host-builder/Dockerfile [hunk 1]
```diff
@@ -6,12 +6,31 @@ ADD bin/start.sh /root/
...
+	python-glanceclient \
+	python-heatclient \
+	python-neutronclient \
+	python-novaclient \
+	python-saharaclient \
+	python-swiftclient \
+	python-troveclient \
+	python-openstackclient && \
+    yum install -y --enablerepo=rhel-7-server-ose-3.1-rpms \
+	ansible-1.9 \
```
Comment (sabre1041) [inline docker/openshift-host-builder/Dockerfile]: Specifically installing ansible is not required as it is a dependency of `atomic-openshift-utils`

**Comment by sabre1041** [comment]
Installed version of ansible is ansible-1.9.4-1.el7aos.noarch

**Comment by sabre1041** [comment]
@oybed Verified Ansible and Openstack tools match the expected version

**Comment by oybed** [comment]
@sabre1041 thanks for running through this so quickly - appreciate it. @etsauer do you want to give it a go as well, or are we good to merge?

**Comment by etsauer** [comment]
@oybed if you guys are good, @sabre1041 can go ahead and merge.

**Pull request merged**